client/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
We can probably nuke this file altogether and add it to the `.gitignore`. Also make sure to run `npm i` after editing the packages.

Reply:
Why, won't we need it for the auth frontend?

Reply:
Oops, mixed it up with `package-lock.json`. It's been a long week